pkg/go/graph/graph.go

```diff
@@ -16,8 +16,10 @@ var ErrBuildingGraph = errors.New("cannot build graph")
 type DrawingDirection bool
 
 const (
+	// DrawingDirectionListObjects is when terminal types have outgoing edges and no incoming edges.
 	DrawingDirectionListObjects DrawingDirection = true
-	DrawingDirectionCheck       DrawingDirection = false
+	// DrawingDirectionCheck is when terminal types have incoming edges and no outgoing edges.
+	DrawingDirectionCheck DrawingDirection = false
 )
```

Comment:
drawingDirection is a private member.  Can we expose this via public method?

Reply:
Sorry I don't understand, you mean you can't access the enum from the outside??

Reply:
Done in https://github.com/openfga/language/pull/354